src/main/java/com/powsybl/openloadflow/network/DiscreteVoltageControl.java

```diff
@@ -56,6 +56,11 @@ public void addController(LfBranch controllerBranch) {
         controllers.add(controllerBranch);
     }
 
+    public boolean removeController(LfBranch controllerBranch) {
```

Comment:
You don't need this kind of function as you will have to merge discrete voltage controls: one discrete voltage control will be totally deleted.